apix-integrationtests/build.gradle

```diff
@@ -13,7 +13,7 @@ subprojects {
     sourceSets {
         main {
             java {
-                srcDir '../../apix-impl/src/integration-test'
+                srcDir '../src/integration-test/java'
```

Comment:
Did you check code navigation is still working in IntelliJ after this change? (This will require a reindexing first otherwise you don't see if it broke)